RSHelpers/RSMemoryReader.cs

```diff
@@ -50,10 +50,10 @@ public RSMemoryReadout DoReadout()
             //If there was string in memory
             if (preview_name != null)
             {
-                //Verify Play_ prefix and _Preview suffix
-                if (preview_name.StartsWith("Play_") && preview_name.EndsWith("_Preview"))
+                //Verify Play_ prefix and _Preview or _Invalid suffix
```

Comment:
I would like a comment to briefly explain that _Invalid is a RSMod replacement to avoid possible future head scratching.